### PR TITLE
Support for OV 2021.4 library

### PR DESCRIPTION
**Pull request by tanmayv25**
These changes would be necessary to support latest OV release 2021.4
Don't merge yet. These changes could be useful when we support OV 2021.4.

### CODE REVIEW

**Comment by CoderHam** [comment]
@tanmayv25 can you please rebase these changes?

**Comment by tanmayv25** [comment]
> @tanmayv25 can you please rebase these changes?

Done

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ questions or report problems in the main Triton [issues
 page](https://github.com/triton-inference-server/server/issues). The backend
 is designed to run models in Intermediate Representation (IR). See [here](https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model.html) for instruction to convert a model to IR format. The backend is implemented using openVINO C++ API. Auto completion of the model config is not supported in the backend and complete `config.pbtxt` must be provided with the model.
 
-## Supported Devices
+F## Supported Devices
```
Comment (tanmayv25) [inline README.md:45]: Typo

**Comment by tanmayv25** [comment]
Closing the pull request as the support was added via PR #32

**Pull request closed without merging**